<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewNewsMessage</class>
 <widget class="QDialog" name="DialogNewNewsMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send new message</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="lineEditTitle"/>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEditMessage"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSend">
       <property name="text">
        <string>Send</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/news/interfaceIcons/newsSend.png</normaloff>:/news/interfaceIcons/newsSend.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
